source/types.ts

```diff
@@ -37,6 +37,10 @@ export interface NextcloudClientInterface extends NextcloudClientProperties {
   checkConnectivity():                                                        Promise<boolean>;
   get(path: string):                                                          Promise<string | Buffer>;
 
+  // Common
+  getFileOrFolderCreator(path: string):                                       Promise<string>;
```

Comment:
You're saying the return type if a Promise, but it could be `null`.

Reply:
Isn't it an async function? If so, a Promise is always returned